packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -276,6 +280,14 @@ class WebView extends StatefulWidget {
   /// [WebViewController.evaluateJavascript] can assume this.
   final PageFinishedCallback onPageFinished;
 
+  /// Invoked when the URL in the browser changed.
+  ///
+  /// While it is possible to see all URL changes in the [navigationDelegate]
+  /// in iOS. This is not the case for Android when JavaScript updates the URL.
```

Comment:
Maybe: 

_While it is possible to see all URL changes in the [navigationDelegate], it's is not the case for Android when JavaScript updates the URL._

Can you please clarify in the comment that the callback works for both iOS & Android (if so)?

Reply:
Yes, I changed it to 

> This is always triggered when the URL changes. Also when this is done via
> JavaScript and no [onPageStarted] or [onPageFinished] events would be fired.

I guess this is most clear of in which case to use it